src/github/push.ts

```diff
@@ -3,8 +3,10 @@ import { Context } from "probot/lib/context";
 import { getCurrentTime } from "utils/webhook-utils";
 import { hasJiraIssueKey } from "utils/jira-utils";
 import { GitHubPushData } from "../interfaces/github";
+import { WebhookContext } from "../routes/github/webhook/webhook-context";
 
-export const pushWebhookHandler = async (context: Context, jiraClient): Promise<void> => {
+export const pushWebhookHandler = async (probotContext: Context, jiraClient, _util, _githubInstallationId: number, webhookContext?: WebhookContext): Promise<void> => {
+	const context = webhookContext ? webhookContext : probotContext;
```

Comment:
We should probably set context based on the `new-webhook-receiver` FF you created.

Reply:
I'm thinking of using FF `new-webhook-receiver` at the start of new webhook receiver flow.  It doesn't make sense to use FF flag here because `webhookContext` will only be populated with new flow. Please create another ticket for FF.